plugins/ingest/src/test/java/org/elasticsearch/ingest/processor/join/JoinProcessorFactoryTests.java

```diff
@@ -32,20 +32,10 @@
     public void testCreate() throws Exception {
         JoinProcessor.Factory factory = new JoinProcessor.Factory();
         Map<String, Object> config = new HashMap<>();
-        Map<String, String> fields = Collections.singletonMap("field1", "-");
-        config.put("fields", fields);
+        config.put("field", "field1");
+        config.put("separator", "-");
         JoinProcessor joinProcessor = factory.create(config);
-        assertThat(joinProcessor.getFields(), equalTo(fields));
-    }
-
-    public void testCreateMissingFields() throws Exception {
-        JoinProcessor.Factory factory = new JoinProcessor.Factory();
-        Map<String, Object> config = new HashMap<>();
-        try {
-            factory.create(config);
-            fail("factory create should have failed");
-        } catch(IllegalArgumentException e) {
-            assertThat(e.getMessage(), equalTo("required property [fields] is missing"));
-        }
```

Comment:
seems like this test should be updated, not removed? test when field or separator are missing?